usecases/ast_eval/evaluate/evaluate_aggregator.go

```diff
@@ -75,12 +75,12 @@ func (a AggregatorEvaluator) Evaluate(ctx context.Context, arguments ast.Argumen
 	// Filters validation
 	var filtersWithType []models.FilterWithType
 	if len(filters) > 0 {
-		for _, filter := range filters {
+		var errs []error
```

Comment:
Use `errs := make([]error, 0, len(filters))` instead to preallocate the slice to the adequate capacity.